README.md

```diff
@@ -1,3 +1,3 @@
 # orcas-phantom-back
 [![Build Status](https://travis-ci.org/atlp-rwanda/orcas-phantom-backend.svg?branch=ch-traviscov-backend)](https://travis-ci.org/atlp-rwanda/orcas-phantom-backend)
-[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/orcas-phantom-backend/badge.svg?branch=ch-traviscov-backend)](https://coveralls.io/github/atlp-rwanda/orcas-phantom-backend?branch=ch-traviscov-backend)
+[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/orcas-phantom-backend/badge.svg?branch=fx-coveralls)](https://coveralls.io/github/atlp-rwanda/orcas-phantom-backend?branch=fx-coveralls)
```

Comment:
Yes, @Malaba6 you are right. But know that it will be updated after the branch has been merged. 
I'm done with it now, you can merge it to see if the badge passes on develop. Thank you.